README.md

```diff
@@ -1,3 +1,11 @@
 # The DMV
 
 This is the starter repo for the BE Mod1 DMV project.
+
+Reflection on Debugging:
+
+1. This time around the first step I took was to look over read the code and get a bit familiar then ran my spec files. From there pulled up each of the 3 spec files that had failing tests and also split screened with the lib files. From there I worked from the bottom up from the list of tests. Felt it was easier to start with debugging one test then moving to two then three until I was fully passing. 
+
+2. I think the hardest thing I found working on code I didnt write was trying to understand what the original author was trying to do with some of the code and navigating is test files. I was a bit confused at first with the added describe line with every test which is something I have not done. 
```

Comment:
Did you find it easier to read through the tests with that added describe line? I find it helps with organizing it blocks.